specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json

```diff
@@ -1248,6 +1278,12 @@
         "responses": {
           "200": {
             "description": "Linked server was successfully deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
>ErrorResponse [](start = 98, length = 13)

The addition of `default` in this PR got flagged as breaking changes. This is a false positive and is not actually a breaking change.